lib/systems/default.nix

```
@@ -25,7 +25,24 @@ let
   inspect = import ./inspect.nix { inherit lib; };
   platforms = import ./platforms.nix { inherit lib; };
   examples = import ./examples.nix { inherit lib; };
-  architectures = import ./architectures.nix { inherit lib; };
+
+  architectures = throw "lib.systems.architectures was deprecated and replaced via the new cpuModel attribute";
```

Deprecated has a specific meaning of "still works but we will remove in the future".
```suggestion
  architectures = throw "architectures was replaced by `cpuModel` attribute";
```

---

Alright, changed.